README.md

```diff
@@ -28,150 +33,30 @@ flake8 picas tests
 pytest tests
 ```
 
-Examples
+Token commands
 ========
 
-## Setting up the examples
```

Comment:
By splitting the Token commands from the Examples, there is now quite some overlap between the sections. This makes it more difficult to follow. I would start with the Examples, and then Token commands.